src/service.ts

```diff
@@ -150,6 +150,13 @@ export class DebugService implements IDebugger, IDisposable {
     Signal.clearData(this);
   }
 
+  /**
+   * is actual kernel state
```

Comment:
This function's name is a little confusing. I'm not sure what it should be called, but I think it should have a more descriptive name.